paper.tex

```diff
@@ -427,16 +427,22 @@ \subsection{Classes}
 It can be mapped to the following EO code, where a class becomes a factory of objects:
 
 \begin{ffcode}
++alias org.eolang.cage
++alias org.eolang.txt.sprintf
++alias org.eolang.io.stdout
++alias org.eolang.hamcrest.assert-that
+
```

Comment:
@ivan-egorov42 I think it would be better to use `Home` form of aliases - `QQ`.

Reply:
Ok, I can change this.